aiohttp/web_exceptions.py

```diff
@@ -410,3 +468,19 @@ class HTTPNotExtended(HTTPServerError):
 
 class HTTPNetworkAuthenticationRequired(HTTPServerError):
     status_code = 511
+
+
+def _initialize_default_reason() -> None:
+    for obj in globals().values():
```

Comment:
How about using
```suggestion
    for obj in HTTPException.__subclasses__():
```
which gets immediate children of the class.

Reply:
Interesting, but we need a recursive iteration to collect all classes

Reply:
Like here? https://stackoverflow.com/a/3862957/595220
Such helper looks quite clean. If we were on Python 3.6 I'd suggest `__init_subclass__()` for such tracking instead...

Reply:
Let's keep the code as is